### PR TITLE
Replace pagination with improved filters for better blockchain exploration

### PR DESCRIPTION
**Pull request by n1tranquilla**
Closes: #646

### CODE REVIEW

**Comment by n1tranquilla** [comment]
@robinbb back to you

File: src/common/constants.rs [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
 pub const BERKELEY_FEATURES_ENABLED: &str = env!("BERKELEY_FEATURES_ENABLED");
-pub const TABLE_RECORD_SIZE: i64 = 100;
-pub const TABLE_DEFAULT_PAGE_SIZE: usize = 10;
-pub const ESTIMATED_ROW_HEIGHT: usize = 48;
-pub const DEFAULT_ESTIMATED_NON_TABLE_SPACE_IN_SECTIONS: usize = 160;
 pub const EPOCH_SLOTS: u16 = 7140;
 pub const DEFAULT_USER_INPUT_DEBOUNCE_INTERNVAL: f64 = 500.0;
 pub const GLOBAL_SEARCH_PLACEHOLDER_TEXT: &str = "Paste -> Enter -> Explore!";
 pub const LIVE_RELOAD_INTERVAL: u64 = 60000;
+pub const TABLE_ROW_LIMIT: i64 = 100;
```
Comment (jhult) [inline src/common/constants.rs:8]: I thought we were going to use 1000

Comment (n1tranquilla) [reply src/common/constants.rs:8]: We can't until we switch to the indexer.

**Review by jhult — APPROVED** [review]
I've tested this locally and it LGTM. Great work @n1tranquilla.

**Pull request merged**